examples/13-1d_holstein/run_afqmc.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+np.random.seed(125)
+from mpi4py import MPI
+
+from ipie.qmc.afqmc import AFQMC
+from ipie.estimators.energy import EnergyEstimator
+from ipie.systems import Generic
+from ipie.addons.eph.hamiltonians.holstein import HolsteinModel
+from ipie.addons.eph.trial_wavefunction.toyozawa import ToyozawaTrial
+from ipie.addons.eph.trial_wavefunction.variational.toyozawa_variational import variational_trial_toyozawa
+from ipie.addons.eph.walkers.eph_walkers import EphWalkers
+from ipie.addons.eph.propagation.holstein import HolsteinPropagatorImportance
+from ipie.qmc.options import QMCParams
+
+#System Parameters
+nup = 2
+ndown = 2
+nelec = [nup, ndown]
+
+#Hamiltonian Parameters
+g = 2.
+t = 1.
+w0 = 4.
+nsites = 4
+pbc = True
+
+#Walker Parameters & Setup
+comm = MPI.COMM_WORLD
+nwalkers = 1000 // comm.size
+
+#Setup initial guess for variational optimization
+initial_electron = np.random.random((nsites, nup + ndown))
+initial_phonons = np.ones(nsites) * 0.1
+
+#System and Hamiltonian setup
+system = Generic(nelec)
+ham = HolsteinModel(g=g, t=t, w0=w0, nsites=nsites, pbc=pbc)
+ham.build()
+
+#Variational procedure
+etrial, beta_shift, el_trial = variational_trial_toyozawa(
```

Comment:
Why are you settings this and not just recomputing it as a check?

Reply:
(this is a good unit test.)

Reply:
No, computing them in the script is fine. I was saying the trial wavefunction should recompute the energy given your optimized orbitals / parameters. This is a useful sanity check when running jobs as we typically generate trials externally so it's a form of handshake to make sure everything is formatted / interpreted correctly. Here you're just setting them rather than recalculating them via the trial.

The unit test is separate and is just a simple assertion that building a trial wavefunction from your optimized parameters reproduces the variational energy.